app/views/histories/index.html.erb

```diff
@@ -2,125 +2,179 @@
-        <ul>
-          <li><%= link_to 'Prime Ministers', historic_appointments_path('past-prime-ministers') %></li>
-          <li><a href="/government/history/past-chancellors">Chancellors</a></li>
-          <li><a href="/government/history/past-foreign-secretaries">Foreign Secretaries</a></li>
-        </ul>
-      </div>
+    <div class="govuk-grid-column-one-half image">
+      <span class="aspect-ratio-3:2">
+        <%= image_tag 'how-gov-works/Churchill_01.jpg', alt: "", loading: "lazy" %>
+      </span>
+    </div>
+    <div class="govuk-grid-column-one-half">
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Notable people",
+        id: "notable-peoples",
+        margin_bottom: 3,
+      } %>
+
+      <p class="govuk-body">You can find out about the life and achievements of those who have led the government as Prime Minister, alongside those who have held the roles of Chancellor and Foreign Secretary.</p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>
+          <%= link_to 'Prime Ministers', historic_appointments_path('past-prime-ministers'), class: "govuk-link" %>
+        </li>
+        <li>
+          <%= link_to "Chancellors", "/government/history/past-chancellors", class: "govuk-link" %>
+        </li>
+        <li>
+          <%= link_to "Foreign Secretaries", "/government/history/past-foreign-secretaries", class: "govuk-link" %>
+        </li>
+      </ul>
     </div>
   </div>
-</div>
 
-<div class="block government-buildings">
-  <div class="inner-block floated-children">
-    <hr class="keyline">
-    <div class="column">
-      <div class="content">
-        <h2 id="government-buildings">Government buildings</h2>
-        <p>Number 10 Downing Street is the best known government building, but there are others that  play a role in the day-to-day business of government and have had a significant role in our history.</p>
-        <p>You can learn more about the background and take tours of several of the buildings used by government on Whitehall and around the UK.</p>
-        <ul>
-          <li><a href="/government/history/10-downing-street">10 Downing Street</a></li>
-          <li><a href="/government/history/11-downing-street">11 Downing Street</a></li>
-          <li><a href="/government/history/king-charles-street">King Charles Street (FCO)</a></li>
-          <li><a href="/government/history/lancaster-house">Lancaster House (FCO)</a></li>
-          <li><a href="/government/publications/history-of-the-ministry-of-defence">Ministry of Defence, Whitehall</a></li>
-          <li><a href="/government/history/1-horse-guards-road">1 Horse Guards Road</a></li>
-          <li><a href="/hillsborough-castle">Hillsborough Castle</a></li>
-        </ul>
-      </div>
+  <div class="govuk-grid-row government-buildings">
+    <div class="govuk-grid-column-full">
+      <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--l">
```

Comment:
I don't know how this bit of govuk-frontend works, but does this HR need to be in a grid column? Could it be moved a few lines up to between the wrapping govuk-grid-row and the previous one?

Reply:
The `hr` doesn't have any left or right margin - so is wider than the full width column. I could add the margin with the override classes - but that's not a good path to take. Doing what you propose means it will overhang each side:

![image](https://user-images.githubusercontent.com/1732331/74450239-b26f4c00-4e75-11ea-9f26-2dd4803a7a60.png)

My thought was either adding more CSS to add a margin to the left and right of the `hr`; or adding more HTML to wrap it in a full width column. 

Not much between either option to be honest - so I chose adding the column in the template because I think that's clearer in showing what's being done, and doesn't add anything new.

Reply:
Cool, that's fine. Tricky reviewing when you can't see it, eh. Happy to look over this when you're deploying if you like.